src/react-components/ory/sections/oidc-section.tsx

```diff
@@ -5,19 +5,23 @@ import { SelfServiceFlow } from "../helpers/types"
 import { hasOidc } from "../helpers/utils"
 
 export const OIDCSection = (flow: SelfServiceFlow): JSX.Element | null => {
+  const hasOidcTraits =
+    filterNodesByGroups({
+      nodes: flow.ui.nodes,
+      groups: "oidc",
+      withoutDefaultGroup: true,
+      excludeAttributes: "submit",
+    }).length > 0
+
   return hasOidc(flow.ui.nodes) ? (
     <div className={gridStyle({ gap: 32 })}>
-      {filterNodesByGroups({
-        nodes: flow.ui.nodes,
-        groups: "oidc",
-        withoutDefaultGroup: true,
-      }).some((node) => node.messages.length > 0) && (
```

Comment:
it only rendered if there were error messages in the top level and not in any of the fields...